components/TextFields/tests/unit/ContainedInputView/MDCBaseTextFieldTests.m

```diff
@@ -184,4 +184,67 @@ - (void)testClearButtonRectForBounds {
   XCTAssertTrue(CGRectEqualToRect(actualClearButtonFrame, expectedClearButtonFrame));
 }
 
+- (void)testFloatingLabelColorAccessors {
+  // Given
+  CGRect textFieldFrame = CGRectMake(0, 0, 130, 40);
+  MDCBaseTextField *textField = [[MDCBaseTextField alloc] initWithFrame:textFieldFrame];
+  UIColor *floatingLabelColor = [UIColor redColor];
+
+  // When
+  [textField setFloatingLabelColor:floatingLabelColor forState:UIControlStateNormal];
+  [textField setFloatingLabelColor:floatingLabelColor forState:MDCTextControlStateEditing];
+  [textField setFloatingLabelColor:floatingLabelColor forState:UIControlStateDisabled];
+  [textField setNeedsLayout];
+  [textField layoutIfNeeded];
+
+  // Then
+  XCTAssertEqualObjects(floatingLabelColor,
+                        [textField floatingLabelColorForState:UIControlStateNormal]);
+  XCTAssertEqualObjects(floatingLabelColor,
+                        [textField floatingLabelColorForState:MDCTextControlStateEditing]);
+  XCTAssertEqualObjects(floatingLabelColor,
+                        [textField floatingLabelColorForState:UIControlStateDisabled]);
+}
```

Comment:
I was also expecting a test for the fallback logic of these getters. If normal is set and you ask for the others it uses normal. 

Reference test for MDCButton does this here:

https://github.com/material-components/material-components-ios/blob/e3784afe259055679479f09b2e92324b61cf7fe0/components/Buttons/tests/unit/MDCButtonTests.m#L482